libbpf-cargo/src/btf/types.rs

```diff
@@ -183,6 +184,20 @@ pub struct BtfTypeTag<'a> {
     pub type_id: u32,
 }
 
+#[derive(Debug)]
+pub struct BtfEnum64Value<'a> {
+    pub name: &'a str,
+    pub value: u64,
+}
+
+#[derive(Debug)]
+pub struct BtfEnum64<'a> {
+    pub name: String,
```

Comment:
Why do we need an owned string here, as opposed to `&'a str`?

Reply:
Enum could be anonymous, so we name it.

```rust
let name = match Self::get_btf_str(string_table, t.name_off as usize)? {
    "" => {
        self.anon_count += 1;
        format!("{}{}", ANON_PREFIX, self.anon_count)
    }
    n => n.to_string(),
};
```